src/FondOfSpryker/Zed/CustomerPriceList/Business/CustomerPriceListBusinessFactory.php

```diff
@@ -18,4 +20,12 @@ public function createCustomerExpander(): CustomerExpanderInterface
     {
         return new CustomerExpander($this->getRepository());
     }
+
+    /**
+     * @return \FondOfSpryker\Zed\CustomerPriceList\Business\Model\CustomerReaderInterface
+     */
+    public function createCustomerReader(): CustomerReaderInterface
+    {
+        return new CustomerReader($this->getRepository());
```

Comment:
@SpiderDan98 why CustomerReader? Which kind of object will you get?

Reply:
A PriceListCollectionTransfer

Reply:
@SpiderDan98 the correct name for this class is PriceListReader or CustomerPriceListReader. If you want to get a customer, the name CustomerReader will be correct.